src/main/java/com/google/sps/data/UserInfo.java

```diff
@@ -0,0 +1,17 @@
+package com.google.sps.data;
+
+public class UserInfo {
+  // User's unique identifier in Firebase.
+  private final String uid;
+  // User's address that is editable.
+  private String address;/*  */
+  // This will be set to true the first time the user makes a delivery request. This way,
+  // only previous signed up couriers will be able to see "See journeys" page.
+  private boolean isCourier;
+
+  public UserInfo(String uid, String address, boolean isCourier) {
+    this.uid = uid;
+    this.address = address;
+    this.isCourier = isCourier;
+  }
+}
```

Comment:
You could also create a UserInfo object only with uid and have separate methods for `setAddress(String address)` and `setIsCourier(boolean isCourier)`. This can be useful if you want to change the value of the address or isCourier without creating a new object.